### PR TITLE
update sedfile for harbor helm chart

### PR DESCRIPTION
**Pull request by vincentni**

### CODE REVIEW

File: projects/goharbor/harbor/helm/sedfile.template [hunk 1]
```diff
@@ -1,4 +1,4 @@
-s/^version: 1.9.1$/version: ${IMAGE_TAG}/
+s/^version: 1.11.1$/version: ${IMAGE_TAG}/
```
Comment (ivyostosh) [inline projects/goharbor/harbor/helm/sedfile.template:1]: Will this work? So you don't need to hardcode it every time
`s/version: .*$/version: ${IMAGE_TAG}/`

Comment (vincentni) [reply projects/goharbor/harbor/helm/sedfile.template:1]: It gets passed by our script

**Pull request merged**